### ISSUE TITLE
Overfitting problem while implementing in Camelyon16

### ISSUE THREAD

**Issue by pzSuen** [szc19990412/TransMIL#13]
Hello,

I randomly select 90% and 10% of official train data as train-set and val-set. The test-set is official provided.
But the gap between training and inference is big.
Have you encountered this problem?

![image](https://user-images.githubusercontent.com/43957007/165949209-d07e3be5-fd80-4f86-9e9c-a85a3c00b098.png)

**Comment by Bingchao-Zhao** [comment]
Hello, 
@pzSuen 
I'm having some problems reproducing transmil. As mentioned in the article, the feature of patch is obtained from resnet50, but the output of resnet50 backbone is 2048, while the input of _fc1 of transmil is 1024. how can I get the right input vector?

**Comment by pzSuen** [comment]
> 

As in the paper, you can find these related content from the paper "Data-efficient and weakly supervised computational pathology on whole slide images".

**Comment by Bingchao-Zhao** [comment]
@pzSuen 
Thanks for the reminder, you have solved a puzzle that has been bothering me for a long time.

**Comment by marvinyan080** [comment]
@pzSuen Have you been able to reproduce the results reported in the paper?

**Comment by Bingchao-Zhao** [comment]
I was able to get a performance close to the reported after extracting features using the network provided by CLAM.



***@***.***
 
From: marvinyan080
Date: 2022-08-09 13:28
To: szc19990412/TransMIL
CC: Bingchao Zhao; Comment
Subject: Re: [szc19990412/TransMIL] Overfitting problem while implementing in Camelyon16 (Issue #13)
@pzSuen Have you been able to reproduce the results reported in the paper?

**Comment by szc19990412** [comment]
Hi.  Since the CAMELYON16 dataset is a public dataset for a competition, and all the results are obtained in the public test set, so we only reported the results with the highest folds.  Indeed, the mean and variance should be used as a more reasonable criterion, and we will pay more attention to this issue in the future.
